.github/workflows/tests.yml

```diff
@@ -37,13 +37,15 @@ jobs:
       deleted_formulae: ${{ steps.formulae-detect.outputs.deleted_formulae }}
     steps:
       - name: Set up Homebrew
+        if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'CI-published-bottle-commits')
```

Comment:
Should we move these three statements up here?

https://github.com/Homebrew/homebrew-core/blob/a236345fdfbb37465edc0b5a0ad1a51dcbd84423/.github/workflows/tests.yml#L28

Reply:
Ah, yes, I think we should. In that way `setup_tests` (and subsequently, `tests`) may also be skipped due to `needs: tap_syntax`. But that's fine.

https://github.com/Homebrew/homebrew-core/blob/1828a10760f818891bd199ca8a057c330354d558/.github/workflows/tests.yml#L51-L56